app/controllers/pair_requests_controller.rb

```diff
@@ -3,6 +3,7 @@ class PairRequestsController < ApplicationController
 
   # GET /pair_requests or /pair_requests.json
   def index
+    @pair_request = PairRequest.new
```

Comment:
mentioned below but change to `@new_pair_request`

Reply:
yeah, agreed, I thought about that after I went out. Kind of whipped it up, but that's an easy and reasonable switch to make.